projects/knora-ui/src/lib/viewer/values/text-value/text-value-as-html/text-value-as-html.component.html

```diff
@@ -0,0 +1,8 @@
+<span [formGroup]="form">
+    <mat-form-field class="large-field">
+        <input matInput [formControlName]="'textValue'" class="value" placeholder="Text value" type="text" [readonly]="mode === 'read'">
```

Comment:
Probably we don't need a form field with an input if we are just going to inject HTML into the DOM.